server/routes/nl/helpers.py

```diff
@@ -418,9 +418,7 @@ def abort(error_message: str,
       'config': {},
       'context': escaped_context_history,
       'failure': error_message,
-      'userMessage': {
-          'msg': error_message
-      },
+      'userMessages': [error_message]
```

Comment:
Can this impact non-explore clients talking to the API - un, nodejs?

https://github.com/datacommonsorg/website/blob/99b87eb1d7180a840f93af512f283db2a586864f/experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/components/search/Search.tsx#L281

Reply:
ah, you're right, maybe we should return both userMessage and userMessages then & eventually deprecate userMessage? What is the best practice when it comes to updating a response like this?

Reply:
Yes, have both, change client, then drop.

@dwnoble can point you to the repo where the client code is.

Reply:
updated to have both & a TODO to remove userMessage once UN client is updated